pkg/kritis/review/validating_transport_test.go

```diff
@@ -0,0 +1,102 @@
+/*
+Copyright 2019 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package review
+
+import (
+	"encoding/base64"
+	"reflect"
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/grafeas/kritis/pkg/kritis/apis/kritis/v1beta1"
+	"github.com/grafeas/kritis/pkg/kritis/attestation"
+	"github.com/grafeas/kritis/pkg/kritis/metadata"
+	"github.com/grafeas/kritis/pkg/kritis/testutil"
+	"github.com/grafeas/kritis/pkg/kritis/util"
+)
+
+func TestValidatingTransport(t *testing.T) {
```

Comment:
Let's split success tests from error tests here.

Reply:
I skimmed this from TestHasValidAttestations (review_test.go) and as far as I can tell there are no error tests here.  So I don't think there's anything to split.  Do you want me to add error tests?

Reply:
I initially read the tests similar to `no valid sig` as error tests, but you're right. Yes please, let's add error tests here, this is generally the pattern we'd like to follow in our codebase (and we're not there yet).

Reply:
Done.